### ISSUE TITLE
Operator >>.~ incorrectly formatted

### ISSUE THREAD

**Issue by CPardi** [fsprojects/fantomas#291]
I have found that the operator >>.~ is incorrectly changed to >> after formatting. The example below

```fs
module Scratch

let (>>.~) (g : int) (h : int) : int = g + h
let output = 2 >>.~ 3
```

results in the following:

```fs
module Scratch

let (>>) (g : int) (h : int) : int = g + h
let output = 2 >> 3
```

This happens when using the online formatting tool and the version 2.8.0 Nuget package.

Thanks,
Chris

**Comment by CPardi** [comment]
Hi jindraivanek, thanks for looking into this.  
It's interesting that the bug stems from FSharp.Compiler.Service. I use the Rider IDE, which is built on top of FSharp.Compiler.Service, and in newer versions has trouble highlighting files when I use this operator. I know nothing about the FSharp.Compiler.Service but this being a bug there would explain why this operator causes problems for both Fantomas and Rider.

**Comment by nojaf** [comment]
Should be fixed in 2.8.1.
